src/lava/magma/runtime/runtime.py

```diff
@@ -31,9 +31,49 @@
 from lava.magma.compiler.node import NodeConfig
 from lava.magma.core.run_conditions import AbstractRunCondition
 
+"""Defines a Runtime which takes a lava executable and a pluggable message 
```

Comment:
Should this be here or in the class docstring?

Reply:
These are file docstrings